doc/man/8/rbdmap.rst

```diff
@@ -1,128 +0,0 @@
-:orphan:
```

Comment:
But man page section 8 is for system administration commands. Since we have removed the `rbdmap` executable, we should also remove this page, or it may cause confusion. Maybe we can add another rbdmap man page to section 5 (File formats and conventions)?

Reply:
My suggestion is to add a note at the beginning of the manpage that the tool has been removed and the man is still here just to provide information about /etc/ceph/rbdmap format which is still supported for compatibility reasons although deprecated.

But I don't have a strong opinion here and will be actually fine with any solution.